resend/actions/submitEmail.ts

```diff
@@ -0,0 +1,34 @@
+import { AppContext } from "../mod.ts";
```

Comment:
I would prefer a more idiomatic actions. So instead of

resend.actions.submitEmail()

I would go for

resend.actions.emails.send()

So you can "send emails" using resend app.